test/module/irohad/validation/query_execution.cpp

```diff
@@ -726,21 +737,21 @@ class GetAccountAssetsTransactionsTest : public QueryValidateExecuteTest {
     txs_observable = getDefaultTransactions(account_id, asset_id);
   }
 
-  rxcpp::observable<Transaction> getDefaultTransactions(
-      const std::string &creator_id, const std::string &asset_id) {
+  rxcpp::observable<std::shared_ptr<shared_model::interface::Transaction>>
+  getDefaultTransactions(const std::string &creator_id,
```

Comment:
It looks the same as https://github.com/hyperledger/iroha/pull/966/files#diff-3a520ff855c891c9d7353c3271254cb2R563 , please remove this method